roxie/ccd/ccdqueue.cpp

```diff
@@ -2371,6 +2680,14 @@ class RoxieSocketQueueManager : public RoxieReceiverBase
                     break;
                 }
             }
+#ifdef NEW_IBYTI
+            delayed.checkExpired(
+                            msTick(),
```

Comment:
minor: could common this msTick() up with the one above.

Reply:
The one above is the other side of the readtms...